tools/python_api/src_cpp/py_query_result.cpp

```diff
@@ -137,6 +137,13 @@ py::object PyQueryResult::convertValueToPyObject(const Value& value) {
         auto blobBytesArray = blobStr.c_str();
         return py::bytes(blobBytesArray, blobStr.size());
     }
+    case LogicalTypeID::UUID: {
+        kuzu::common::int128_t result = value.getValue<kuzu::common::int128_t>();
+        std::string int128_string = kuzu::common::Int128_t::ToString(result);
+        py::object Decimal = py::module_::import("decimal").attr("Decimal");
```

Comment:
Python has a [native type for UUID](https://docs.python.org/3/library/uuid.html). I think it would be better to convert UUIDs into the native type instead of returning as decimal.